src/solver/conv_direct_naive_conv_bwd.cpp

```diff
@@ -102,6 +81,40 @@ ConvSolution ConvDirectNaiveConvBwd::GetSolution(const ConvolutionContext& ctx)
     int c_per_group = c / group;
     int k_per_group = k / group;
 
+    int block_size = 256;
+    int grid_size  = 1;
+    if(ctx.IsLayoutDefault())
+    {
+        grid_size = n * c;
+    }
+    else if(ctx.IsLayoutNHWC())
+    {
+        if(ctx.Is2d())
+            grid_size = group * n * hi;
+        else
+            grid_size = group * n * di;
+    }
+    else
+        MIOPEN_THROW("Unsupported layout");
+
+    KernelInfo kernel;
+
+    kernel.kernel_file = ConvDirectNaiveConvKernelFile(ctx);
+    kernel.kernel_name = ConvDirectNaiveConvKernelName(ctx);
+    kernel.g_wk.clear();
+
+    kernel.g_wk.push_back(grid_size * block_size);
+    kernel.g_wk.push_back(1);
+    kernel.g_wk.push_back(1);
+    kernel.l_wk.clear();
+    kernel.l_wk.push_back(block_size);
+    kernel.l_wk.push_back(1);
+    kernel.l_wk.push_back(1);
+
+    kernel.comp_options = ConvDirectNaiveConvCompileOption(ctx);
+
+    MIOPEN_LOG_I2(kernel.kernel_file + ":" + kernel.kernel_name);
```

Comment:
Please remove this. We should have general machinery that prints Solutions (including filenames and kernel names) when necessary.

Reply:
OK